File: andrew035/questao17.py
n = float(input("Informe a quantidade em 1 centavos: "))
s = float(input("Informe a quantidade em 5 centavos: "))
d = float(input("Informe a quantidade em 10 centavos: "))
c = float(input("Informe a quantidade em 25 centavos: "))
e = float(input("Informe a quantidade em 50 centavos: "))
r = float(input("Informe a quantidade em 1 real: "))

q = n * 0.01
w = s * 0.05
t = d * 0.10
y = c * 0.25
u = e * 0.50
o = r * 1.00

reais = q + w + t + y + u + o

print("A quantidade de dinheiro em reais é: " + str(reais))

File: naelson022/programacao_basica/Questao17.java
import java.util.Scanner;

public class Questao17 {
    public static void main (String [] args ) {
        Scanner s;
        double w;
        double u;
        s = new Scanner (System.in);
        System.out.println("calcule de dinheiro");
        u = 0;
        System.out.print(" informe a quantidade de moedas 1  centavos");
        w = s.nextInt ();
        u = u + w * 0.01;

        System.out.print(" informe a quantidade de moedas 5  centavos");
        w = s.nextInt ();
        u = u + w * 0.05;

        System.out.print(" informe a quantidade de moedas 10  centavos");
        w = s.nextInt ();
        u = u + w * 0.10;

        System.out.print(" informe a quantidade de moedas 25  centavos");       w = s.nextInt ();
        u = u + w * 0.25;

        System.out.print(" informe a quantidade de moedas 50  centavos");
        w = s.nextInt ();
        u = u + w * 0.50;

        System.out.print(" informe a quantidade de moedas 1 real");
        w = s.nextInt ();
        u = u + w * 1.00;
        System.out.println(" o dinheiro economizado eh " + u);

    }
}
